client/app/pages/dashboards/components/DashboardHeader.jsx

```diff
@@ -41,7 +41,9 @@ function DashboardPageTitle({ dashboardOptions }) {
             ignoreBlanks
           />
         </h3>
-        <img src={dashboard.user.profile_image_url} className="profile-image" alt={dashboard.user.name} />
+        <Tooltip title={dashboard.user.name} placement="bottom">
```

Comment:
FWIW: This event doesn't work on mobile since there's no "hover" event. Not a big deal but maybe something to address down the road.

Reply:
On mobile you can click on it to see the tooltip.

Reply:
This doesn't work on the latest iOS via iPhone XS. Does work on iPad.

Reply:
It worked for me on iPhone X. 😮

Reply:
I bet that even if this didn't work it wouldn't impact many users 🤷‍♂️